pkg/sigv4/sigv4.go

```
@@ -60,6 +53,16 @@ type Config struct {
 	Region        string
 }
 
+func (c Config) asSha256() string {
+	h := sha256.New()
+	_, err := h.Write([]byte(fmt.Sprintf("%v", c)))
+	if err != nil {
+		return ""
```

Nit. Not sure in what way this could fail, but feels like the error should be returned rather than swallowed.